pkg/webhook/validator.go

```diff
@@ -460,12 +463,22 @@ func ValidatePolicy(ctx context.Context, namespace string, ref name.Reference, c
 }
 
 func ociSignatureToPolicySignature(ctx context.Context, sigs []oci.Signature) []PolicySignature {
-	// TODO(vaikas): Validate whether these are useful at all, or if we should
-	// simplify at least for starters.
-	ret := []PolicySignature{}
+	ret := make([]PolicySignature, 0, len(sigs))
 	for _, ociSig := range sigs {
 		logging.FromContext(ctx).Debugf("Converting signature %+v", ociSig)
-		ret = append(ret, PolicySignature{Subject: "PLACEHOLDER", Issuer: "PLACEHOLDER"})
+
+		if cert, err := ociSig.Cert(); err == nil && cert != nil {
+			ce := cosign.CertExtensions{
+				Cert: cert,
+			}
```

Comment:
This is extracted directly from how `cosign` outputs this information during verification today.